.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I just copied all these templates, if you'd like to change something, please let me know.